jobs/job.go

```diff
@@ -150,16 +150,7 @@ func (j *Job) UnmarshalJSON(data []byte) error {
 }
 
 // ReleaseJobParams are the params for a release job
-type ReleaseJobParams struct {
-	ServiceSpecs []flux.ServiceSpec
-	ImageSpec    flux.ImageSpec
-	Kind         flux.ReleaseKind
-	Excludes     []flux.ServiceID
-
-	// Backwards Compatibility, remove once no more jobs
-	// TODO: Remove this once there are no more jobs with ServiceSpec, only ServiceSpecs
-	ServiceSpec flux.ServiceSpec
-}
+type ReleaseJobParams flux.ReleaseSpec
```

Comment:
Probably not. Thought it might be cleaner to have a alias here so the param types are all consistent.